components/prism-core.js

```diff
@@ -101,6 +101,46 @@ var _ = {
 				default:
 					return o;
 			}
+		},
+
+		/**
+		 * Returns the script element that is currently executing.
+		 *
+		 * This does __not__ work for line script element.
+		 *
+		 * @returns {HTMLScriptElement | null}
+		 */
+		currentScript: function () {
+			if (typeof document === 'undefined') {
+				return null;
+			}
+			if ('currentScript' in document) {
+				return document.currentScript;
+			}
+
+			// IE11 workaround
+			// we'll get the src of the current script by parsing IE11's error stack trace
+			// this will not work for inline scripts
+
+			try {
+				throw new Error();
```

Comment:
Do we need to throw the Error to get a stack trace?

Reply:
How else do you wanna do it?
Also, the stack trace is non-standard, so it's IE11 specific.

Reply:
Oh, I just couldn't remember if just creating the error object captures the stack trace or if it needs to be thrown.

Reply:
That's what you meant. I tested this and IE11 adds the `stack` property after its thrown.
Don't know how other browsers handle this though.

Reply:
Fair enough. This is fine, since it's an IE11-specific workaround.